test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -1,774 +1,760 @@
 describe('Visual graph screen validation', function () {
```

Comment:
What's the matter with this file? Seems like it's re formatted and now it's not obvious what's changed.